src/algorithms/post/election_post.md

```diff
@@ -148,12 +148,12 @@ For i=0; i < challNumber; i++:
     ranHash = H(ticket, i)
     ranIndex = HashToInt(ranHash) mod len(PowerTable)
```

Comment:
Imo we must fix this biased way of generating a random index.

Reply:
do you have a suggestion?